### ISSUE TITLE
How to convert the depth values of the generated stereo data into corresponding disparity values?

### ISSUE THREAD

**Issue by kaillerC94** [HanLingsgjk/UnifiedGeneralization#5]
How to convert the depth values of the saved stereo data into corresponding disparity values? 
For the stereo dataset generated from the scene reconstructed by [fabiotosi92/NeRF Superior Deep Stereo: A novel paradigm for collecting and generating stereo training data using neural rendering (GitHub)](https://nerfstereo.github.io/), where are the corresponding information such as baseline and focal to get final disparity value?

**Comment by HanLingsgjk** [comment]
In render_stereo.py line 380: writeFlowKITTI(os.path.join(output_filename_flow, fileid), flowf1.permute(1,2,0).detach().cpu().numpy())
What is stored here is the disparity value.

you can read it by:
def readFlowKITTI(filename):
    flow = cv2.imread(filename, cv2.IMREAD_ANYDEPTH|cv2.IMREAD_COLOR)
    flow = flow[:,:,::-1].astype(np.float32)
    flow, valid = flow[:, :, :2], flow[:, :, 2]
    flow = (flow - 2**15) / 64.0
    return flow, valid

and flow(:,:,0) is the stereo

**Comment by kaillerC94** [comment]
Thank you for this reply. But what i want to know is about the depth saved by writeDDDCNERF (render_stereo.py, line 382). I know that depth1 is the depth info of left_image of the stereo pair. But how to convert this depth value into disparity value between the stereo pair.

**Comment by kaillerC94** [comment]
Ohh, you're right, really grateful~
